requirements.txt

```diff
@@ -36,7 +36,7 @@ qrcode==5.3
 reportlab==3.3.0
 requests==2.20.0
 zeep==3.1.0
-vatnumber==1.2
+python-stdnum==1.11
```

Comment:
Need to check if this is the correct version

Reply:
Why not 1.12? See https://pypi.org/project/python-stdnum/1.12/

Reply:
@Yenthe666 I think it needs to be the one of the last debian/ubuntu stable by default

Reply:
Ahh yes that makes sense. Would need a check, it's probably lower than the latest :)

Reply:
@julienlegros In ubuntu 18.04 LTS, the version is 1.8, so would we need to put 1.8 here then?  I would actually prefer to use a newer version as Yenthe suggests as it supports certain things that are important to us.  (but then we probably need to change below too)

Reply:
and it is sure, it is not python3-stdnum?

Reply:
Unless we need a specific feature belonging to a more recent version, the preferred approach is to require what's in the latest Debian, i.e. `python-stdnum==1.10`. Ubuntu 20.04 will ship with at least 1.12.

Reply:
@hpr-odoo @jva-odoo Can you still adapt this to 1.10 as Julien said?

Reply:
@jco-odoo , changed to 1.10. Thank you